autoload/verilog_systemverilog.vim

```diff
@@ -587,6 +587,11 @@ function! verilog_systemverilog#VerilogErrorFormat(...)
   if (l:tool == "ncverilog")
     " Based on https://github.com/vhda/verilog_systemverilog.vim/issues/88
     set errorformat=%.%#:\ \*%t\\,%.%#\ \(%f\\,%l\|%c\):%m
+    set errorformat+=%.%#\ *%t\\,%.%#(%f\\,%l\|%c):\ %m
```

Comment:
I don't think we need this one. We just need to have `\ %#` in the line above. I see now that I have been using `\*` without needing to... too many regular expressions in my head! :P
I'll include this fix and re-submit, such that you can use `git rebase` over my last changes.